src/main/java/mate/academy/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.bookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
prefer unchecked exceptions over checked
```suggestion
public class RegistrationException extends RuntimeException {
```